test/index.test.js

```diff
@@ -121,6 +121,43 @@ test.cb('test method without arguments', (t) => {
     })
 })
 
+test.cb('test method without arguments - options attached to file', (t) => {
+  const { metalsmith } = t.context
+
+  metalsmith
+    .use((files) => {
+      files['example.jpg'].sharp = {
+        namingPattern: '{dir}negated{ext}',
```

Comment:
Lets use a different naming scheme here, you are actually overwriting the file negated file from the other test. You might even later on test the file from the other test and your's didn't work at all 😱

Reply:
Good point. 👍